fpdf/__init__.py

```diff
@@ -40,7 +40,6 @@
     "FlexTemplate",
     "TitleStyle",
     "ViewerPreferences",
-    "HTMLMixin",
```

Comment:
Removing this class here will have some unwanted effect on user code, that will probably break when they start to use the next `fpdf2` release.
At this point we do not want users code to break, but only to raise a deprecation warning.
We should find a way to trigger a `DeprecationWarning` when the following code construct is used:
```python
class PDF(FPDF, HTMLMixin):
    pass
```